dev/diagrams/on-app-booting-hook.md

```diff
@@ -0,0 +1,41 @@
+# On App Booting Hook
+
+<!-- TOC -->
+
+- [Use Is App Booted Hook](#use-is-app-booted-hook)
+  - [Overview](#overview)
+
+<!-- /TOC -->
+
+## Overview
+This is a basic activity diagram for the useIsAppBootedHook.
+
+
+```plantuml
+
+@startuml
+
+title useIsAppBootedHook()
+
+participant "End User" as user
+
+box "FE" #Lightblue
+  participant "Component" as fe
+  participant "Hook: useIsAppBooted" as hook
+box
+
+activate user
+  user -> fe: open app
+  activate fe
+    fe -> hook: useIsAppBooted
+    activate hook
+      break if there is no authPrep or the user is not signed in
+        hook -> hook: handleSignOutApp()
```

Comment:
```suggestion
        hook -> hook: handleSignOutApp()
        fe <- hook: Returns nothing
```
The current flow still makes the `` happens as the red line.
![image](https://github.com/ajktown/docs/assets/53258958/c20328bc-b748-4c81-9ade-cf95fd39a88b)


You need to make sure that the break ends at the component level!
![image](https://github.com/ajktown/docs/assets/53258958/c2fbe3f0-5863-49ce-a3d6-1e875bcb0eeb)